department-of-veterans-affairs/caseflow
# appeals-13259 setup decision reasons in cavc dashboard

Resolves [13259](https://vajira.max.gov/browse/APPEALS-13259?workflowName=VA+Caseflow+Sandbox+Story+Enabler+Workflow&stepId=20)

### Description
Setup Decision Reasons accordion in the cavc dashboard page.

### Acceptance Criteria

### Testing Plan
1. Go to a cavc dashboard page
2. For any issue, set disposition to either "**Reversed**" or "**Vacated and Remanded**"
3. Observe that the Decision Reason section appears
4. Click on the Decision Reason section and observe that it expands to show the parent checkboxes
5. Check if each checkbox behaves correctly:
**(Behavior of checkboxes when checked. The numbers represent the order of the checkboxes):**
Parent Checkboxes that should not expand when checked: **1,3,4,5,7,8,11,13,14,15,17**
Parent Checkboxes that should expand with selection basis when checked: **6**
Parent Checkboxes that should expand with child checkboxes when checked: **2,9,10,12,16,18**
Child Checkboxes that should expand with selection basis when checked: **9.1, 10.1, 12.1-12.4, 16.1-16.4, 18.7**
Child Checkboxes that should not expand when checked: **2.1-2.2, 9.2, 10.2, 18.1-18.6**
6. For parent checkboxes, observe that a numerical value appears next to the header "Decision Reasons" in parenthesis and corresponds to the number of parent checkboxes that are checked.
### User Facing Changes

BEFORE
<img width="1333" alt="before" src="https://user-images.githubusercontent.com/111081469/220432798-e3e1bb96-5e5d-4526-9234-6e9530c52667.png">

AFTER
<img width="1333" alt="after" src="https://user-images.githubusercontent.com/111081469/220432813-6feaa32b-2240-4cca-b1da-b280888ed14b.png">
